### PR TITLE
Update network_increase_bandwidth.md

### PR DESCRIPTION
**Pull request by cebe**
Follow-up to https://github.com/lxc/lxd/pull/11392#issuecomment-1441511468 and https://github.com/lxc/lxd/issues/11198

### CODE REVIEW

File: doc/howto/network_increase_bandwidth.md [hunk 1]
```diff
@@ -41,4 +41,8 @@ Complete the following steps to increase the network bandwidth on the LXD host:
 
 ## Increase the transmit queue length on the instances
 
-You must also change the `txqueuelen` value for all Ethernet interfaces in your instances, in the same way as described for the LXD host.
+You must also change the `txqueuelen` value for all Ethernet interfaces in your instances.
+There are two alternative ways to do this:
+
+- apply the same changes as described above for the LXD host, or
```
Comment (tomponline) [inline doc/howto/network_increase_bandwidth.md:47]: @ru-fu do we need a capital for first letter here?

File: doc/howto/network_increase_bandwidth.md [hunk 2]
```diff
@@ -41,4 +41,8 @@ Complete the following steps to increase the network bandwidth on the LXD host:
...
+- set the `queue.tx.length` device option on the instance profile or configuration.
```
Comment (ru-fu) [inline doc/howto/network_increase_bandwidth.md:48]:
```suggestion
To do this, use one of the following methods:

- Apply the same changes as described above for the LXD host.
- Set the `queue.tx.length` device option on the instance profile or configuration.
```

File: doc/howto/network_increase_bandwidth.md (hunk 1 above)
Comment (ru-fu) [reply doc/howto/network_increase_bandwidth.md:47]: Yes, and we shouldn't use an "or" at the end of a list item. I added a suggestion. :)

File: doc/howto/network_increase_bandwidth.md (hunk 2 above)
Comment (cebe) [reply doc/howto/network_increase_bandwidth.md]: applied. Thanks!

**Comment by cebe** [comment]
should I rebase this to fix the "signoff" build error?

**Review by ru-fu — APPROVED** [review]
Doc looks good now, thanks!

But you should rebase this to one commit.

**Comment by stgraber** [comment]
Going to do the rebase myself so we can merge this.

**Pull request merged**